libs/api/metadata-converter/src/lib/xml-utils.ts

```diff
@@ -448,3 +489,29 @@ export function renameElements(
   doReplace(rootElement)
   return rootElement
 }
+
+export function readGeometry(el: XmlElement): Geometry {
+  const gmlString = xmlToString(el)
```

Comment:
So, to avoid the GML namespace stuff in `xmlToString` you can do the following here:

```suggestion
  const elWithNs = new XmlElement(
    el.name,
    {
      ...el.attributes,
      'xmlns:gml': 'http://www.opengis.net/gml/3.2',
      xmlns: 'http://www.opengis.net/gml/3.2',
    },
    el.children
  )
  const gmlString = xmlToString(elWithNs)
```

Essentially, we make a clone of the element to read and we set both the `gml` namespace and the default namespace to GML 3.2. This way, we're (almost?) sure that OL will be able to parse the geometry.

What do you think?

Reply:
Oh, actually another even simpler solution:
```suggestion
  const gmlString = xmlToString(createDocument(el))
```

This way an actual document will be created out of the node, including all the namespaces.